apps/crn-server/test/fixtures/discover.fixtures.ts

```diff
@@ -3,117 +3,8 @@ import {
   DiscoverResponse,
   UserResponse,
 } from '@asap-hub/model';
-import { FetchDiscoverQuery } from '../../src/autogenerated-gql/graphql';
 import { appName, baseUrl } from '../../src/config';
 
-const getSquidexGraphqlDiscoverMembers = (prefix = '') => [
-  {
-    id: `${prefix}uuid-members-1`,
-    created: '2020-10-15T17:55:21Z',
-    lastModified: '2020-10-15T17:55:21Z',
-    version: 42,
-    flatData: {
-      avatar: [
-        {
-          id: `${prefix}uuid-1`,
-        },
-      ],
-      email: `${prefix}john@example.com`,
-      firstName: `${prefix}John`,
-      lastModifiedDate: '2020-10-15T17:55:21Z',
-      lastName: `${prefix}Doe`,
-      institution: `${prefix}ASAP`,
-      jobTitle: `${prefix}Job title`,
-    },
-  },
-  {
-    id: `${prefix}uuid-members-2`,
-    created: '2020-10-14T17:55:21Z',
-    lastModified: '2020-10-15T17:55:21Z',
-    version: 42,
-    flatData: {
-      email: null,
-      lastModifiedDate: '2020-10-15T17:55:21Z',
-      lastName: `${prefix}Do`,
-      firstName: `${prefix}John`,
-      institution: `${prefix}ASAP`,
-      jobTitle: `${prefix}Job title`,
-      avatar: [
-        {
-          id: `${prefix}uuid-2`,
-        },
-      ],
-    },
-  },
-];
-
-const squidexGraphqlDiscoverFlatData = (): NonNullable<
-  FetchDiscoverQuery['queryDiscoverContents']
->[number]['flatData'] => ({
-  training: [
-    {
-      id: 'uuid-training-1',
-      created: '2020-09-24T11:06:27.164Z',
-      flatData: {
-        title: 'Title',
-        text: '<p>Content</p>',
-        link: 'https://hub.asap.science',
-        linkText: 'ASAP Training',
-        shortText: 'Short text',
-        thumbnail: [
-          {
-            id: 'thumbnail-uuid-1',
-          },
-        ],
-      },
-    },
-  ],
-  pages: [
-    {
-      id: 'uuid-pages-1',
-      created: '2020-09-24T11:06:27.164Z',
-      lastModified: '2020-10-15T17:55:21Z',
-      version: 42,
-      flatData: {
-        title: 'Title',
-        text: '<p>Content</p>',
-        link: 'https://hub.asap.science',
-        linkText: 'ASAP Hub',
-        shortText: 'Short text',
-      },
-    },
-    {
-      id: 'uuid-pages-2',
-      created: '2020-09-24T11:06:27.164Z',
-      lastModified: '2020-10-15T17:55:21Z',
-      version: 42,
-      flatData: {
-        title: 'Title',
-        text: '<p>Content</p>',
-        link: 'https://hub.asap.science',
-        linkText: 'ASAP Hub',
-        shortText: 'Short text',
-      },
-    },
-  ],
-  members: getSquidexGraphqlDiscoverMembers(),
-  membersTeam: [
-    {
-      id: 'uuid-team-1',
-    },
-  ],
-  scientificAdvisoryBoard: getSquidexGraphqlDiscoverMembers('sab-'),
-  aboutUs: '<p>content</p>',
-});
-
-export const getSquidexGraphqlDiscover = () => ({
-  id: 'ec3086d4-aa64-4f30-a0f7-5c5b95ffbcca',
-  created: '2020-09-23T16:34:26.842Z',
-  lastModified: '2021-05-14T14:48:46Z',
-  version: 42,
-  flatData: squidexGraphqlDiscoverFlatData(),
-});
-
 const getSquidexGraphqlDiscoverMembersResponse = (
```

Comment:
Maybe we could rename this?

```suggestion
const getGraphqlDiscoverMembersResponse = (
```